tensorboard/plugins/image/summary.py

```diff
@@ -129,7 +131,35 @@ def pb(name, images, max_outputs=3, display_name=None, description=None):
   return summary
 
 
+class _TensorFlowPNGEncoder(object):
```

Comment:
Nit: Png not PNG, which is what go/java-style recommends, and is the convention I've been following. Python style doesn't specifically say, but I think it's kind of Googley.